api/orchestration.go

```diff
@@ -142,6 +145,14 @@ func WithRecursive(recursive bool) TerminateOptions {
 	}
 }
 
+// WithRecursivePurge configures whether to purge all sub-orchestrations created by the target orchestration.
+func WithRecursivePurge(recursive bool) PurgeOptions {
```

Comment:
I would recommend renaming `WithRecursive` above to `WithRecursiveTerminate`

Reply:
Done